kafka/vendor/six.py

```diff
@@ -70,7 +70,9 @@ def __len__(self):
         else:
             # 64-bit
             MAXSIZE = int((1 << 63) - 1)
-        del X
+
+        # Don't del it here, cause with gc disabled this "leaks" to garbage
+        # del X
```

Comment:
this one is a vendored library, and I really would prefer fixing upstream. otherwise I'm likely to forget and overwrite when I import a newer version. Have you checked if this is fixed already in six?